modules/sfp_abusech.py

```diff
@@ -58,8 +58,45 @@
 }
 
 class sfp_abusech(SpiderFootPlugin):
-    """abuse.ch:Investigate,Passive:Reputation Systems::Check if a host/domain, IP or netblock is malicious according to abuse.ch."""
```

Comment:
This should remain in place for now. We will only remove it once the meta objects are fully in place everywhere and we've updated the code that relies upon its existence.